Packages/com.unity.inputsystem/InputSystem/Devices/Sensor.cs

```diff
@@ -77,13 +81,56 @@ public static FourCC kFormat
             get { return new FourCC('L', 'A', 'A', 'C'); }
         }
 
-        [InputControl] public Vector3 acceleration;
+        [InputControl(processors = "CompensateDirection")] 
+        public Vector3 acceleration;
 
         public FourCC GetFormat()
         {
             return kFormat;
         }
     }
+    
+    public class CompensateDirectionProcessor : IInputControlProcessor<Vector3>
+    {
+        public Vector3 Process(Vector3 value, InputControl control)
+        {
+            if (!InputConfiguration.CompensateSensorsForScreenOrientation)
+                return value;
+            
+            var rotation = Quaternion.identity;
+            switch (Screen.orientation)
+            {
+                case ScreenOrientation.PortraitUpsideDown: rotation = Quaternion.Euler(0, 0, 180); break;
+                case ScreenOrientation.LandscapeLeft: rotation = Quaternion.Euler(0, 0, 90); break;
+                case ScreenOrientation.LandscapeRight: rotation = Quaternion.Euler(0, 0, 270); break;
```

Comment:
Stuff like this really needs comments.

Reply:
You should have commented before merging the PR :D